framework/contracts/account/manager/src/contract.rs

```diff
@@ -206,6 +206,16 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg) -> M
                         cw_ownable::Action::AcceptOwnership => update_governance(deps.storage)?,
                         _ => vec![],
```

Comment:
We should clear the cache state (proposed governance) when the `RenounceOwnership` variant is called.